src/standalone/Vegetation/radiation.jl

```diff
@@ -88,15 +90,19 @@ $(DocStringExtensions.FIELDS)
 Base.@kwdef struct TwoStreamParameters{
     FT <: AbstractFloat,
     G <: AbstractGFunction{FT},
+    APAR <: Union{FT, ClimaCore.Fields.Field},
+    TPAR <: Union{FT, ClimaCore.Fields.Field},
+    ANIR <: Union{FT, ClimaCore.Fields.Field},
+    TNIR <: Union{FT, ClimaCore.Fields.Field},
 }
     "PAR leaf reflectance (unitless)"
-    α_PAR_leaf::FT
+    α_PAR_leaf::APAR
```

Comment:
I believe these should all be the same type, e.g. 
```
α_PAR_leaf::F
τ_PAR_leaf::F
α_NIR_leaf::F
τ_NIR_leaf::F
```
where `{F <: Union{FT, ClimaCore.Fields.Field}}`. I dont think we need to support a mix of floats and fields - either the parameters are scalars or they are fields. And forcing them to have the same type also makes sure the fields are defined on the same space.

I think this should be true for all of these parameter structs. so if there are already Fields you introduced to this struct, it should also have type `F`